alts/src/main/java/io/grpc/alts/internal/TsiFrameHandler.java

```diff
@@ -169,6 +168,10 @@ public void flush(final ChannelHandlerContext ctx) throws GeneralSecurityExcepti
       // protection framing.
       return;
     }
+
+    // There is something to write, we need a frame protector now.
```

Comment:
Is it possible to repro this in a test case?  The Google internal version of this code has a similar problem and I think it would be good to avoid a recurrence.